apps/stripe/src/modules/stripe/stripe-payment-intents-api.ts

```diff
@@ -21,11 +22,24 @@ export class StripePaymentIntentsApi implements IStripePaymentIntentsApi {
   }
 
   async createPaymentIntent(args: {
-    // TODO: narrow type here
-    params: Stripe.PaymentIntentCreateParams;
+    stripeMoney: StripeMoney;
+    intentParams: Pick<
+      Stripe.PaymentIntentCreateParams,
+      "automatic_payment_methods" | "payment_method_options"
+    >;
+    idempotencyKey: string;
```

Comment:
for better code management lets move params to dedicated object and keep it near interface